package/MDAnalysis/coordinates/DMS.py

```diff
@@ -70,7 +70,9 @@ def get_coordinates(self, cur):
         particles = cur.fetchall()
         return [(p['x'], p['y'], p['z']) for p in particles]
 
-    def get_particle_by_columns(self, cur, columns=['x', 'y', 'z']):
+    def get_particle_by_columns(self, cur, columns=None):
```

Comment:
Ah, I'm on holidays and haven't had time to carefully look at these. However, I'll disagree with @orbeckst that Cody did a good job here. It'd be cleaner and more concise to replace the list with a tuple (which is immutable and fulfills the task just as well).